plugins/net_plugin/net_plugin.cpp

```diff
@@ -789,6 +858,108 @@ namespace eosio {
             ( "_lport", local_endpoint_port );
          return mvo;
       }
+
+      // security group management
+      //
+   private:
+      std::optional<chain::account_name> participant_name_;
+      std::atomic<bool> participating_{true};
+
+   public:
+      /** @brief  Returns the optional name of the participant */
+      inline auto participant_name() { return participant_name_; }
+      /** @brief Set the participating status */
+      inline void set_participating(bool status) { 
+         if (my_impl->ssl_enabled)
+            participating_.store(status, std::memory_order_relaxed); 
+      }
+      /** returns true if connection is in the security group */
+      inline bool is_participating() const { 
+         if (my_impl->ssl_enabled)
+            return participating_.load(std::memory_order_relaxed);
+         
+         return true;
```

Comment:
Why returning true when ssl is not enabled?

Reply:
because everything i participating when ssl is not enabled.  That gets us the current behavior.